protool/__init__.py

```diff
@@ -65,6 +65,24 @@ def profile_type(self) -> ProvisioningType:
 
         raise Exception("Unable to determine provisioning profile type")
 
+    @property
+    def developer_certificates(self) -> List[OpenSSL.crypto.X509]:
+        """Returns developer certificates as a list of PyOpenSSL X509."""
+        dev_certs: List[OpenSSL.crypto.X509] = []
+        raw_cert_items: List[str] = cast(List[str], self._contents.get("DeveloperCertificates"))
+
+        try:
+            iter(raw_cert_items)
```

Comment:
As far as I can tell, you are doing this to check to see if you got `None` back from line 72? If so, just add an extra parameter to the `.get` call of `[]` as the default.